INSTALL.md

```diff
@@ -97,3 +108,25 @@ For cross compile, you may want to setup the toolchain something like below:
 
 This assumes you already installed the cross-built `libelf` on the sysroot
 directory.
+
```

Comment:
"Or else you can build it from source (please see below) or use it on a different path using `--with-elfutils=<PATH>`."